tidy3d/components/medium.py

```diff
@@ -976,17 +1050,40 @@ def _eps_dataset_eps_inf_greater_no_less_than_one_sigma_positive(cls, val, value
                     "Permittivity at infinite frequency at any spatial point "
                     "must be no less than one."
                 )
+
+            if modulation is not None and modulation.permittivity is not None:
+                if np.any(eps_real.values - modulation.permittivity.max_modulation <= 0):
```

Comment:
just to double check my understanding is correct: in principle this could be relaxed a bit by computing max modulation at each spatial point? But this would require either checking that eps_real/sigma and space modulation are provided on the same grid, or performing interpolation, and probably not worth doing anyway

Reply:
Yeah, probably slow as well.